user_tools/src/spark_rapids_tools/utils/util.py

```diff
@@ -135,6 +139,7 @@ def gen_app_banner() -> str:
 *     \____/____/\___/_/       /_/  \____/\____/_/____/            *
 *                                                                  *
 *                                      Version. {c_ver}            *
+*                                      {tool_mode_note}            *
```

Comment:
Currently, this is hardcoded for the mode `Development` or when it's empty. 

Can we use string formatters like `{:<24}` for flexible alignment?

For example, Line 142 might get misaligned:
```
*                                      {tool_mode_note}            *
```